modules/ROOT/pages/install-a-bonita-bpm-cluster.adoc

```diff
@@ -36,14 +36,14 @@ Ensure that these ports are opened in all the nodes of your cluster.
 [#create_init_bonita_db]
 === Create and initialize the database for Bonita Platform
 
-In this step you will create and initialize the database for the Bonita Platform cluster using the  xref:bonita-bpm-platform-setup.adoc[platform setup tool].
+In this step you will create and initialize the database for the Bonita Platform cluster using the  xref:bonita-platform-setup.adoc[platform setup tool].
 When done you will have a database with all tables created and with a table `CONFIGURATION` containing all configuration required for the cluster to start.
 
 . Ensure that you meet the  xref:hardware-and-software-requirements.adoc[requirements].
 . xref:database-configuration.adoc#database_creation[Create the database].
 . In case you use  xref:define-and-deploy-the-bdm.adoc[Business data],  xref:database-configuration.adoc#database_creation[create a database for the Business Data].
 . Download a Bonita  xref:tomcat-bundle.adoc[Tomcat bundle] and unzip it at some place of your choice.
```

Comment:
at some place of your choice. => in your chosen folder. / in a folder of your choice.

Reply:
Done